packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.styles.ts

```diff
@@ -165,14 +166,20 @@ export function getStyles(props: ICoachmarkStyleProps): ICoachmarkStyles {
       {
         position: 'absolute',
         top: '50%',
-        left: '50%',
         transform: 'translate(-50%, -50%)',
         width: '0px',
         height: '0px',
         borderRadius: '225px',
         borderStyle: 'solid',
         opacity: '0'
       },
+      getRTL()
```

Comment:
hmm; this should have translated automatically

Reply:
is it the transform that needs to be flipped? right and left do flip but transforms aren't currently.

Reply:
You are correct. After playing around with it I was able to get the correct transform for RTL. I also noticed that the beak for "Right bottom/center/top Edge" was not visible so I changed that as well

![Screen Shot 2019-09-16 at 12 25 51 PM](https://user-images.githubusercontent.com/13246181/64988042-6ba60780-d87f-11e9-8bdc-04e29a9cb307.png)